trussworks/react-uswds
# feat: Add aria attributes to RangeInput

# Summary

Adds aria-min, aria-max, and aria-valuenow labels.  Also adds optional min and max as input props.

## Related Issues or PRs

closes [issue #1542](https://github.com/trussworks/react-uswds/issues/1542)

## How To Test

Tests should reflect added min/max props and corresponding aria values.
To test the value manually, run storybook (`yarn storybook`) and try out the different sliders with the developer tools --> Element open.  You should see `aria-min` and `aria-max` values.  When moving the slider, you should see the `aria-valuenow` changing.
